src/domain/session/room/ComposerViewModel.js

```diff
@@ -0,0 +1,81 @@
+import {ViewModel} from "../../ViewModel.js";
+
+export class ComposerViewModel extends ViewModel {
+    constructor(roomVM) {
+        super();
+        this._roomVM = roomVM;
+        this._isEmpty = true;
+        this._replyId = null;
+        this._replyVM = null;
+    }
+
+    setReplyingTo(entry) {
+        const newId = entry?.id || null;
+        const changed = this._replyId !== newId;
```

Comment:
you could use `this._replyVM.internalId` (see SimpleTile) here and compare it with `entry.asEventKey().toString()`, so you can avoid the `_replyId` field altogether.